engine/src/main/xsd/hibernate-validation-configuration-2.0.xsd

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is a bold move :).

We'll have to wait for Gunnar for that (he's on vacation). I'm not sure we want to extend the schema for Hibernate specific features.

Wondering if it wouldn't be better to use a `<property>`, especially considering it's such a niche feature and it fits well.

Reply:
:) yeah ... I wasn't sure about this as well - if it's a right way to do it. And also about that comment on `validation-custom-config.xml` it's commented out as the test failed and I didn't have time to figure out what happened...

Reply:
@gsmet I've removed the schema, so hope that Gunnar won't see that :). I changed the code to use the property and if the name of the property is OK, I think I could add some paragraph to documentation on how to set the `ScriptEvaluatorFactory` why it's needed and the ways it can be done.